ops/charm.py

```diff
@@ -297,6 +297,20 @@ def restore(self, snapshot):
             self.unit = None
 
 
+class RelationCreatedEvent(RelationEvent):
+    """Represents the `relation-created` hook from Juju.
```

Comment:
Is it """ or '''?  I've seen both in recent PRs.

Reply:
I will stick to `"""` around docstrings for consistency and according to https://www.python.org/dev/peps/pep-0257/#id15.